bpo-43705: Document that SyntaxError's offsets are 1-indexed (#25153)
python/cpython: Doc/library/exceptions.rst

   or :func:`eval`, or when reading the initial script or standard input
   (also interactively).

   The :func:`str` of the exception instance returns only the error message.

   .. attribute:: filename

      The name of the file the syntax error occurred in.

   .. attribute:: lineno

>       Which line number in the file the error occurred in. Note that this is
>       1-indexed, the first line in the file would have a ``lineno`` of 1.

Reviewer: ```suggestion
      Which line number in the file the error occurred in. This is
      1-indexed: the first line in the file has a ``lineno`` of 1.
```

   .. attribute:: offset

>       The column in the line where the error occurred. Note that this is
>       1-indexed, the first character in the line has an ``offset`` of 1.

Reviewer: ```suggestion
      The column in the line where the error occurred. This is
      1-indexed: the first character in the line has an ``offset`` of 1.
```
